lib/share_progress/facebook_variant.rb

```diff
@@ -0,0 +1,23 @@
+require 'share_progress'
+require 'share_progress/variant'
+
+module ShareProgress
+  class FacebookVariant < Variant
+    attr_accessor :facebook_title, :facebook_thumbnail, :facebook_description
+
+    def initialize(facebook_title:, facebook_description:, facebook_thumbnail:, button:)
+      @facebook_title = facebook_title
```

Comment:
do we actually want to make all the fields required? it seems like maybe we should be able to instantiate with nothing